git_machete/docs.py

```diff
@@ -182,6 +200,40 @@
           <b>--c, --checkout-my-github-prs</b>     Checkout your open PRs into local branches.
           <b>-y, --yes</b>                         Don't ask for confirmation when deleting branches from git.
     """,
+    "config": f"""
+        <b>Config keys: </b>
+        - `machete.github.(remote,organization,repository)`:
```

Comment:
```suggestion
        - `machete.github.{remote,organization,repository}`:
```

Parens `()` are uncommon for expansion in CLI docs in general; braces `{}` are better suited as they evoke [shell brace expansion](https://virtuslab.github.io/tips/#shell/shell-brace-expansion)